tensorflow/c/experimental/gradients/BUILD

```diff
@@ -95,3 +109,37 @@ filegroup(
         "//tensorflow/python:__pkg__",
     ],
 )
+
+cc_library(
+    name = "grad_test_helper",
+    testonly = True,
+    srcs = ["grad_test_helper.cc"],
+    hdrs = ["grad_test_helper.h"],
+    visibility = [
+        "//tensorflow:internal",
+    ],
+    deps = [
+        "//tensorflow/c/eager:gradient_checker",
+        "//tensorflow/c/eager:gradients_util",
+        "//tensorflow/core:test",
+        "//tensorflow/core:test_main",
+    ],
+)
+
+tf_cuda_cc_test(
+    name = "nn_grad_test",
+    size = "small",
+    srcs = [
+        "nn_grad_test.cc",
+    ],
```

Comment:
We need to add `    args = ["--heap_check=local"],` back.

That will suppress the memory leak. I will file a bug to clean that up.

Reply:
Done